configure.ac

```diff
@@ -267,7 +267,7 @@ AS_VAR_IF([ac_cv_have_decl_EXT2_IOC_GETFLAGS], [yes],
                     [Define to 1 if you have a working EXT2_IOC_GETFLAGS])])
 
 AC_CHECK_HEADERS([inttypes.h io.h langinfo.h limits.h])
-AC_CHECK_HEADERS([linux/fiemap.h linux/fs.h linux/magic.h linux/types.h])
+AC_CHECK_HEADERS([linux/fiemap.h linux/fs.h linux/version.h linux/magic.h linux/types.h])
```

Comment:
We try to keep these sorted:  linux/version.h goes after linux/types.h.

I suggest just adding a new line

AC_CHECK_HEADERS([linux/version.h])